README.md

```diff
@@ -228,7 +228,7 @@ The `python-domino` client comes bundled with an Operator for use with airflow a
 To install its dependencies, when installing the package from github add the `airflow` flag to extras with pip.
 
 ```
-pip install -e git+https://github.com/dominodatalab/python-domino.git@master#egg=domino[airflow]
+pip install -e git+https://github.com/dominodatalab/python-domino.git@1.0.4#egg=python-domino[airflow]
```

Comment:
@dknupp , you mean you played around with different suffixes and egg names? What exactly does not work?
To answer your question, no I don't have any experience with this but I could try.

Generally speaking, I am concerned about having a specific version of python domino hardcoded anywhere, in environments in particular: you may inadvertently get stuck with an old version whereas previously you would automatically get the latest one.

But it would be good to know that such an option is available (referring to a specific version if that's what you want) so we still need to figure out something that would work.

Reply:
I tried it on a brand new centos instance, and after some trial and error (`yum` seems to expect python 2 but we want python 3 so I had to keep 2 installations of python alongside each other) I reproduced the error in https://dominodatalab.atlassian.net/browse/DOM-30534?atlOrigin=eyJpIjoiNDI4MGNhN2FjMGJhNGI4N2IwODBjN2RhNDU0NjAxMGUiLCJwIjoiamlyYS1zbGFjay1pbnQifQ and when running `pip3 install -e git+https://github.com/dominodatalab/python-domino.git@master#egg=domino[airflow]` and had a successful installation when running `pip3 install -e git+https://github.com/dominodatalab/python-domino.git@master#egg=python-domino[airflow]`.